build.gradle

```diff
@@ -95,16 +94,31 @@ publishing {
 
     repositories {
         maven {
+            def releasesRepoUrl = 'https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/'
+            def snapshotsRepoUrl = 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+            url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
             name = "OSSRH"
-            url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+            // url = "https://s01.oss.sonatype.org/content/repositories/"
             credentials {
-                username = System.getenv('OSSRH_USERNAME')
-                password = System.getenv('OSSRH_PASSWORD')
+                username = 'VaibhavTechsophy'
+                password = '^V3DjU7PHjh7Fda'
             }
         }
     }
 }
 
+nexusPublishing {
+    repositories {
+        sonatype {
+            //only for users registered in Sonatype after 24 Feb 2021
+            nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
+            snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
+            username = 'VaibhavTechsophy'
```

Comment:
again same thing, please review the diff before pushing the code to open source repos.